core/src/main/resources/jenkins/appearance/AppearanceGlobalConfiguration/index.jelly

```diff
@@ -26,31 +26,51 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form">
   <l:layout title="${%Appearance}" permissions="${app.MANAGE_AND_SYSTEM_READ}" type="one-column">
     <j:set var="readOnlyMode" value="${!app.hasPermission(app.MANAGE)}"/>
-    <l:main-panel>
-      <l:app-bar title="${%Appearance}"/>
+    <j:set var="pluginsUrl" value="${rootURL}/manage/pluginManager/available?filter=UI Themes" />
+    <j:set var="hasPlugins" value="${it.hasPlugins()}" />
 
-      <div class="behavior-loading">
-        <l:spinner text="${%Loading}"/>
-      </div>
+    <l:main-panel>
+      <l:app-bar title="${%Appearance}">
+        <j:if test="${hasPlugins}">
+          <a href="${pluginsUrl}" class="jenkins-button">
+            <l:icon src="symbol-plugins" />
+            ${%findPlugins}
```

Comment:
what about removing this from theme manager? it looks strange to have this twice:

![image](https://github.com/user-attachments/assets/b0253ca3-d28d-478a-a1c8-c07f1b80e3fb)